modules/display.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable class-methods-use-this */
+import { Storage } from './storage.js';
+
+const storage = new Storage();
+
+export class Display {
+  constructor() {
+    return null;
+  }
+
+  displayBooks() {
+    const myBooks = storage.fetchBooks();
+    myBooks.forEach((element, index) => {
+      const str = `"${element.book}" by ${element.author}`;
+      document.querySelector('#all-books').innerHTML += `
+      <tr>
+        <td>${str}</td>
+        <td><button type="button" id="book${index}" class="remove">Remove</button></td>
+      </tr>
+      `;
+    });
+  }
```

Comment:
- [ ] [OPTIONAL] it would be better if you use es6 `arrow function` instead of `normal method` in order to practice working with both